### PR TITLE
PA-2353, 2354

### PR DESCRIPTION
**Pull request by jdtoombs**
This PR includes:

- formatting of the fiscal year for financial values in building/land review sections
- removal of validation checkmark on location field (in the review section for building/land)

image

### CODE REVIEW

File: frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx [hunk 1]
```diff
@@ -284,7 +288,8 @@ export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
                   disabled={editInfo.valuation}
                 />
                 <Input
-                  field="data.financials.0.netbook.fiscalYear"
+                  field="netbookYearDisplay"
```
Comment (jdtoombs) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx:291]: Is this ok to have a dummy field for display only? Or should I make a separate component that does not require field etc

File: frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx [hunk 2]
```diff
@@ -12,13 +12,14 @@ import {
 } from 'components/common/form';
```
Comment (jdtoombs) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx:12]: Woops changing assessed back

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.tsx [hunk 3]
```diff
@@ -43,7 +44,14 @@ const LinkButton = styled.span`
 `;
 
 export const LandReviewPage: React.FC<any> = (props: IReviewProps) => {
+  const withNameSpace: Function = useCallback(
```
Comment (Fosol) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.tsx:47]: What's the value of this?  Is it just to make line 54 shorter?  Seems like a util that would be more performant and less duplication if you just did this `applyNamespace(props.nameSpace, 'financials.0.netbook.year')`.

**Review by Fosol — APPROVED** [review]

**Review by agalbrai — APPROVED** [review]
Looks good!

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.tsx (hunk 3 above)
Comment (jdtoombs) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.tsx:47]: Yeah and to keep consistent with the rest of our code, I know the name spaces can get a bit nested for certain fields (buildings, financials) so this helps control it through the parent components. But yes I agree, I'll make a util and include in next PR - we use this a lot!

File: frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx [hunk 4]
```diff
@@ -32,6 +33,7 @@ interface IReviewProps {
 
 export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
   const formikProps = useFormikContext();
+  const netBookYear = getIn(formikProps.values, 'data.financials.0.netbook.fiscalYear');
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx:36]: Let's try and use withNamespace as much as possible rather then hardcoding in the object paths.

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx (hunk 4 above)
Comment (jdtoombs) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx]: Fixed!

**Pull request merged**